tests/test_runner.py

```diff
@@ -48,10 +49,13 @@ def test_invalid_frigg_file_format(self, mock_build_settings):
         """
         self.assertRaises(SystemExit, Runner, False, False)
 
+    @mock.patch('builtins.open' if six.PY3 else '__builtin__.open',
```

Comment:
I think it would be nicer to put the string in `OPEN_MODULE` and use `@mock.patch(OPEN_MODULE, side_effect=lambda *args, **kwargs: FileIO('coverage_report'))`